name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/AccountListView.java

```diff
@@ -818,4 +813,10 @@ private void updateChart(Account account)
         }
     }
 
+    private void deleteAccount(Account account) {
```

Comment:
Just FYI - I inlined that 3 line method as lambda function for readability reasons. No big thing, but I was feeling that a three line method somewhere else requires me to scroll and look up what it does and the lambda just shows in place. And I left the 5 line method you introduced in ClientSideEditor... 🤔